engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/VariableInstanceRestServiceInteractionTest.java

```diff
@@ -177,7 +179,7 @@ public void testGetSingleVariableInstanceSerialized() {
       .body("executionId", equalTo(builder.getExecutionId()))
       .body("caseInstanceId", equalTo(builder.getCaseInstanceId()))
       .body("caseExecutionId", equalTo(builder.getCaseExecutionId()))
-      .body("taskId", equalTo(builder.getTaskId()))
+      .body("batchId", equalTo(builder.getBatchId()))
```

Comment:
Is there a specific reason you removed the "taskId"?

Reply:
Nice catch! This change was accidentally introduced.